import-service/serverless.ts

```diff
@@ -80,6 +88,32 @@ const serverlessConfiguration: AWS = {
           },
         },
       },
+      catalogItemsQueue: {
```

Comment:
as far as I understood, queue should be defined in products service. Because products service is responsible for storage of products and for operations on these products. And the queue is the entrypoint to that service. So queue should be created there and its arn should be exported as output